src/components/Card/CardToolbar.jsx

```diff
@@ -35,11 +35,14 @@ const propTypes = {
   isExpanded: PropTypes.bool,
   className: PropTypes.string,
   ...omit(CardRangePickerPropTypes, 'onClose'),
+  /** Generates the available time range selection options */
+  timeRangeOptions: PropTypes.objectOf(PropTypes.string),
```

Comment:
Should this be imported from the CardPropTypes so we use that proptypes definition abstraction everywhere?

Reply:
The problem here is that its required in CardRangePicker, but not required here or in Card

Reply:
Ah 👍